src/index.html

```diff
@@ -7,8 +7,67 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" 
+      rel="stylesheet" 
+      type="text/css"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav--bar">
+        <ul class="nav--list">
+          <li class="nav--item">
+            <a href="#" class="nav--link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav--item">
+            <a href="#" class="nav--link">
+              Samsung
+            </a>
+          </li>
+          <li class="nav--item">
+            <a href="#" class="nav--link">
+              Smartphones
+            </a>
+          </li>
+          <li class="nav--item">
+            <a 
+              href="#" 
+              class="nav--link"
+              data-qa="hover"
+            >
+              Laptops & computers
+            </a>
+          </li>
+          <li class="nav--item">
+            <a href="#" class="nav--link">
+              Gadgets
+            </a>
+          </li>
+          <li class="nav--item">
+            <a href="#" class="nav--link">
+              Tablets
+            </a>
+          </li>
+          <li class="nav--item">
+            <a href="#" class="nav--link">
+              Photo
+            </a>
+          </li>
+          <li class="nav--item">
+            <a href="#" class="nav--link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+
+    </header>
```

Comment:
```suggestion
      </nav>
    </header>
```
Don't leave empty lines between parent and child elements